tap_postgres/db.py

```diff
@@ -39,23 +39,23 @@ def fully_qualified_table_name(schema, table):
 
 
 def open_connection(conn_config, logical_replication=False):
+    cfg = {
+        'application_name': 'pipelinewise',
+        'host': conn_config['host'],
+        'dbname': conn_config['dbname'],
+        'user': conn_config['user'],
+        'password': conn_config['password'],
+        'port': conn_config['port'],
+        'connect_timeout': 30
+    }
+
+    if conn_config.get('sslmode'):
+        cfg['sslmode'] = conn_config['sslmode']
```

Comment:
this is not in sync with PPW fastsync [here](https://github.com/transferwise/pipelinewise/pull/531/files).

We want to have `ssl` option in the config but this requires `sslmode` option. Can you please change it to `ssl` to keep it in sync with the fastsync behaviour? Would be nice to accept both `true` (string) and `True` (logical)

Reply:
It dosen't require sslmode option as the init.py converts this form ssl - see ehre https://github.com/transferwise/pipelinewise-tap-postgres/blob/30704dac9c4173beb61a507313be3fd5a5c311d6/tap_postgres/__init__.py#L410